src/rprocessing/applet/BuiltinApplet.java

```diff
@@ -32,6 +32,8 @@ public BuiltinApplet() {
     Session session =
         new SessionBuilder().withDefaultPackages().setPackageLoader(packageLoader).build();
     this.renjinEngine = new RenjinScriptEngineFactory().getScriptEngine(session);
+    this.renjinEngine.put("key", "0");
+    this.renjinEngine.put("keyCode", 0);
```

Comment:
Maybe https://github.com/hawkingrei/Processing.R/blob/0edbae2b4e0c1da39cb77d8fd8c54e8fffb27e7d/src/rprocessing/RLangPApplet.java#L133 is a better place to place these `put` statements.